core/services/chainlink/testdata/config-full.toml

```diff
@@ -3,6 +3,11 @@ InsecureFastScrypt = true
 RootDir = 'test/root/dir'
 ShutdownGracePeriod = '10s'
 
+[AuditLoggerConfig]
+forwardToUrl = "http://localhost:9898"
+headers = []
```

Comment:
Could we include some dummy values to check that everything parses?

Reply:
Since this is setting the config directly I think it bypasses the parsing process.

Reply:
New config parses directly from the TOML files. This file is used to sanity check example values for all of the fields.

Reply:
If the intent is for users to input the encoded format, then the type used in the TOML struct will have to implement [TextMarshaler](https://pkg.go.dev/encoding#TextMarshaler)/[TextUnmarshaler](https://pkg.go.dev/encoding#TextUnmarshaler)